tests/datasource/data_connector/test_configured_asset_filesystem_data_connector.py

```diff
@@ -1568,3 +1571,71 @@ def test_one_year_as_1_data_asset_12_batches(empty_data_context, tmp_path_factor
         "default_configured_data_connector_name": ["report_2018"]
     }
     assert len(data_asset_names["default_configured_data_connector_name"]) == 1
+
+
+def test__get_instantiation_through_instantiate_class_from_config(basic_data_connector):
+    # noinspection PyProtectedMember
+    data_references: list = (
```

Comment:
Handled a `TODO` left in the `test_data_connector.py` file by moving the tests to this file. They all use the `pytest.mark.filesystem` marker.